streams/src/test/java/org/apache/kafka/streams/tests/StreamsStandByReplicaTest.java

```diff
@@ -43,18 +43,25 @@
 public class StreamsStandByReplicaTest {
 
     public static void main(final String[] args) throws IOException {
+        if (args.length < 2) {
+            System.err.println("StreamsStandByReplicaTest are expecting two parameters: propFile, additionalConfigs; but only see " + args.length + " parameter");
+            System.exit(1);
+        }
 
         System.out.println("StreamsTest instance started");
 
-        final String kafka = args.length > 0 ? args[0] : "localhost:9092";
-        final String propFileName = args.length > 1 ? args[1] : null;
-        final String additionalConfigs = args.length > 2 ? args[2] : null;
-
-        final Serde<String> stringSerde = Serdes.String();
+        final String propFileName = args[0];
+        final String additionalConfigs = args[1];
```

Comment:
might be follow up, but can we merge additionalConfigs into the provided propFile ?